AEPMessaging/Sources/MessagingConstants.swift

```diff
@@ -112,6 +118,19 @@ enum MessagingConstants {
                 static let SCHEMA = "schema"
                 static let DATA = "data"
 
+                // MARK: Live Activity
+
+                static let LIVE_ACTIVITY_UPDATE_TOKEN = "liveactivityupdatetoken"
+                static let LIVE_ACTIVITY_PUSH_TO_START_TOKEN = "liveactivitypushtostarttoken"
+                static let LIVE_ACTIVITY_TRACK_START = "liveactivitytrackstart"
+                static let LIVE_ACTIVITY_TRACK_STATE = "liveactivitytrackstate"
```

Comment:
can we follow convention with these strings and keep them `camelCase` like the others?